### PR TITLE
Adding Euclidean distance to WMD

### PR DESCRIPTION
**Pull request by ibrahimsharaf**

### CODE REVIEW

**Comment by marco-c** [comment]
Can you add a test too?

**Comment by ibrahimsharaf** [comment]
> Can you add a test too?

2 more tests for Euclidean, zero and non-zero distance?

**Comment by marco-c** [comment]
> 2 more tests for Euclidean, zero and non-zero distance?

OK!

File: tests/test_crash_similarity.py [hunk 1]
```diff
@@ -86,6 +86,38 @@ def test_wmdistance_zero_distance(self):
 
         self.assertEqual(float('inf'), distance)
 
+    def test_wmdistance_euclidean_non_zero_distance(self):
```
Comment (marco-c) [inline tests/test_crash_similarity.py]: Nit: It would be good to refactor the tests to avoid duplicated code, but it's not a big deal.

**Comment by marco-c** [comment]
Thanks!
